tools/cli/go-whisk-cli/commands/api.go

```diff
@@ -27,6 +29,7 @@ import (
     "../wski18n"
 
     "github.com/fatih/color"
+    "github.com/ghodss/yaml"
```

Comment:
Looks like wskdeploy using a different one? @houshengbo  do you know?
https://github.com/openwhisk/openwhisk-wskdeploy/blob/master/Godeps/Godeps.json#L164
```
		{
			"ImportPath": "gopkg.in/yaml.v2",
			"Rev": "a5b47d31c556af34a302ce5d659e6fea44d90de0"
		}
```